R/validate.R

```diff
@@ -100,5 +100,20 @@
       )
     )
   }
+
+  # Make sure that nowcasts can be generated iteratively, ensuring delay_pmf[1]
+  # !=0 while the triangle is a reporting triangle (NAs all in bottom right)
+  if (is.na(triangle[nrow(triangle), 2]) && delay_pmf[1] == 0) {
```

Comment:
do you want to slightly loosen this check to very nearly 0 or is this enough?

Reply:
Hmmm, I think if it is very nearly 0, that means there was at least a single observation with delay =0 and then I don't think you'd want to consider reindexing